### DISCUSSION TITLE
第5节课作业(22班)

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#404] [作业]
第5课作业地址

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/bca4f9fd-b4ca-41ce-858c-021c000b314e

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_5/note_5.md

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture5.md

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/homework/homework_5.md

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/feihhg83bif9mlpn?singleDoc#

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture5-15fef5979b1343b8a10db9bcd82deffc?pvs=4

**Comment by miku8miku** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson5.md
